src/win32/get-printers.js

```diff
@@ -1,27 +1,48 @@
 "use strict";
 
-const execAsync = require("../utils/exec-file-async");
+const execFileAsync = require("../utils/exec-file-async");
 
 const getPrinters = () => {
   const stdoutHandler = (stdout) => {
-    const result = stdout
-      .trim()
-      .split(/[\r\n]+/)
-      .map((line) => line.trim().split(/\s{2,}/));
-
-    const headers = result[0].reduce((acc, curr, index) => {
-      acc[curr] = index;
-      return acc;
-    }, {});
-
-    return result.slice(1).map((printerData) => ({
-      deviceId: printerData[headers["DeviceID"]],
-      name: printerData[headers["Name"]],
-    }));
+    const printers = [];
+
+    stdout
+      .split(/(\r?\n){2,}/)
+      .map((printer) => printer.trim())
+      .filter((printer) => !!printer)
+      .forEach((printer) => {
+        const printerData = {
+          deviceId: null,
+          name: null,
+        };
+
+        const isValid = printer.split(/\r?\n/).some((line) => {
```

Comment:
`isValid` is duplicated in `get-default-printer.js`. Could you please move this function to `utils` folder?

Reply:
Fixed in e6e7c06.